src/APIFunctions/ErrorLog.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios' // what is this??
+
+export async function getAllErrorLogs () {
+  let allErrorLogs = []
+  await axios
+    .get('api/ErrorLog/getErrorLogs')
+    .then(res => {
+      allErrorLogs = res.data
+    })
+    .catch(err => {
+      allErrorLogs = err
+    })
+  return allErrorLogs
+}
+
+export async function AddErrorLog (newError) {
+  let errorCreated = false
+  const errorToAdd = {
```

Comment:
as mentioned before, `errorToAdd` is identitcal to the parameter `newError`. I would just pass `newError` to the axios call.

Reply:
Take a look at [the schema you wrote](https://github.com/SCE-Development/Core-v4/blob/dev/api/models/ErrorLog.js). I would make the non required fields `string|undefined`

Reply:
remove `errorToAdd` as we are just gonna use the parameter